src/ca65/main.c

```diff
@@ -307,6 +307,14 @@ static void SetSys (const char* Sys)
             NewSymbol ("__OSIC1P__", 1);
             break;
 
+        case TGT_CREATIX_EMUL:
+            NewSymbol ("__CREATIX_EMUL__", 1);
+            break;
+            
+        case TGT_CREATIX_NATIVE:
```

Comment:
I don't know if the emulation setting (EMUL vs. NATIVE) justifies a different target. After all, the target machine is always "creatix". EMUL or NATIVE just changes the behavior of the assembler.